pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -408,3 +398,29 @@ func (o *InstallOptions) Run(ctx context.Context) error {
 	recorder.Eventf("StaticPodInstallerCompleted", "Successfully installed revision %s", o.Revision)
 	return nil
 }
+
+func writeConfig(content []byte, atomic bool, contentDir, filename string) error {
```

Comment:
why do we need atomic bool?

Reply:
because not all files need to be written in that way. The other files are written to a per revision directory, like https://github.com/openshift/library-go/blob/3af072dc814786e3c97f88680db3b9ca6b7cecba/pkg/operator/staticpod/installerpod/cmd.go#L274

we could use `prefixed` arg but that would be a little less explicit.